docs/DevelopmentSetupAndroid.md

```diff
@@ -28,7 +28,12 @@ __IMPORTANT__: Make sure the `ANDROID_HOME` environment variable points to your
         
         export ANDROID_HOME=<path_where_you_unpacked_android_sdk>
 
-  - **On Windows**, go to `Control Panel` -> `System and Security` -> `System` -> `Change settings` -> `Advanced` -> `Environment variables` -> `New`
+  - **On Windows**, go to `Control Panel` -> `System and Security` -> `System` -> `Change settings` -> `Advanced` -> `Environment variables` -> `New`.   
+   
+__NOTE__: You need to restart the Command Prompt (Windows) / Terminal Emulator (Mac OS X, Linux) to apply the new Environment variables.
+
+### Install Git and add it to PATH (windows)  
```

Comment:
### Install Git

  - **On Mac**, if you have installed XCode, Git is already installed, otherwise run the following:

         brew install git

  - **On Linux**, install Git [via your package manager](https://git-scm.com/download/linux).

  - **On Windows**, Download and install [Git for Windows](https://git-for-windows.github.io/). During the setup process, choose "Run Git from Windows Command Prompt", which will add Git to your `PATH` environment variable.